config/config.go

```diff
@@ -574,24 +574,29 @@ func (c *Config) ToRouteSrvOptions() routesrv.Options {
 
 	return routesrv.Options{
 		Address:                            c.Address,
-		SourcePollTimeout:                  time.Duration(c.SourcePollTimeout) * time.Millisecond,
-		WaitForHealthcheckInterval:         c.WaitForHealthcheckInterval,
+		DefaultFiltersDir:                  c.DefaultFiltersDir,
+		KubernetesAllowedExternalNames:     c.KubernetesAllowedExternalNames,
 		KubernetesInCluster:                c.KubernetesInCluster,
 		KubernetesURL:                      c.KubernetesURL,
 		KubernetesHealthcheck:              c.KubernetesHealthcheck,
 		KubernetesHTTPSRedirect:            c.KubernetesHTTPSRedirect,
 		KubernetesHTTPSRedirectCode:        c.KubernetesHTTPSRedirectCode,
 		KubernetesIngressClass:             c.KubernetesIngressClass,
 		KubernetesRouteGroupClass:          c.KubernetesRouteGroupClass,
-		WhitelistedHealthCheckCIDR:         whitelistCIDRS,
 		KubernetesPathMode:                 c.KubernetesPathMode,
 		KubernetesNamespace:                c.KubernetesNamespace,
+		KubernetesEnableEastWest:           c.KubernetesEnableEastWest,
+		KubernetesEastWestDomain:           c.KubernetesEastWestDomain,
 		KubernetesEastWestRangeDomains:     c.KubernetesEastWestRangeDomains.values,
 		KubernetesEastWestRangePredicates:  c.KubernetesEastWestRangePredicates,
 		KubernetesOnlyAllowedExternalNames: c.KubernetesOnlyAllowedExternalNames,
-		KubernetesAllowedExternalNames:     c.KubernetesAllowedExternalNames,
```

Comment:
Not needed?

Reply:
I moved it around to have a lexicographically sorted struct assignment to easier compare the left with the right.

Reply:
It's here https://github.com/zalando/skipper/pull/1873/files#diff-fe44f09c4d5977b5f5eaea29170b6a0748819c9d02271746a20d81a5f3efca17R578